src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Abstractions/IDocumentFilter.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions
+{
+    /// <summary>
+    /// This interface allows creating custom document filters to modify the contents of the OpenApi / Swagger documentation before it is rendered.
+    /// </summary>
+    public interface IDocumentFilter
+    {
+        /// <summary>
+        /// This method is invoked after the <see cref="IDocument"/> has been built and just before it is rendered and returned to the client.
+        /// </summary>
+        /// <param name="document">The generated document.</param>
+        /// <param name="request">The HTTP request.</param>
+        void Apply(OpenApiDocument document, IHttpRequestDataObject request);
```

Comment:
Can we change the method parameter signature to `void Apply(IHttpRequestDataObject request, OpenApiDocument document)` so that it looks consistent with other methods?